src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -48,6 +53,14 @@ public override bool SupportsOnGrab
             }
         }
 
+        public override bool SupportsOnSystemUpdateAvailable
```

Comment:
`SupportsOnSystemUpdateAvailable` makes it sound like an OS update, `SupportsOnUpdateAvailable` would make more sense.